modules/heatpump.py

```diff
@@ -0,0 +1,133 @@
+import logging
+from typing import List
+
+
+class HeatPump:
+    MAX_HEATOUTPUT = 5000  # Maximum heating power in watts
+    BASE_HEATPOWER = 235.0  # Base heating power value
+    TEMPERATURE_COEFFICIENT = -11.645  # Coefficient for temperature
+    COP_BASE = 3.0  # Base COP value
+    COP_COEFFICIENT = 0.1  # COP increase per degree
```

Comment:
This is the docstring syntax for class attributes:

```suggestion
    MAX_HEATOUTPUT = 5000
    """Maximum heating power in watts"""

    BASE_HEATPOWER = 235.0
    """Base heating power value"""

    TEMPERATURE_COEFFICIENT = -11.645
    """Coefficient for temperature"""

    COP_BASE = 3.0
    """Base COP value"""

    COP_COEFFICIENT = 0.1
    """COP increase per degree"""
```